src/lomc.jl

```diff
@@ -236,6 +236,28 @@ function Base.show(io::IO, st::QMCState)
     end
 end
 
+function report_default_metadata!(report::Report, state::QMCState)
+    report_metadata!(report, "Rimu.PACKAGE_VERSION", Rimu.PACKAGE_VERSION)
+    # add metadata from state
+    report_metadata!(report, "laststep", state.laststep)
+    report_metadata!(report, "num_replicas", length(state.replicas))
+    report_metadata!(report, "hamiltonian", state.hamiltonian)
+    report_metadata!(report, "r_strat", state.r_strat)
+    report_metadata!(report, "s_strat", state.s_strat)
+    report_metadata!(report, "τ_strat", state.τ_strat)
+    params = state.replicas[1].params
+    report_metadata!(report, "params", params)
+    report_metadata!(report, "dτ", params.dτ)
+    report_metadata!(report, "step", params.step)
+    report_metadata!(report, "shift", params.shift)
+    report_metadata!(report, "shiftMode", params.shiftMode)
+    report_metadata!(report, "maxlength", state.maxlength[])
+    report_metadata!(report, "post_step", state.post_step)
+    report_metadata!(report, "v_summary", summary(state.replicas[1].v))
+    report_metadata!(report, "v_type", typeof(state.replicas[1].v))
```

Comment:
Would probably be good to do this for all replicas, but we can do it another time. I have some plans to change how replicas work (to facilitate stuff like ACX) in the future anyway.